### ISSUE TITLE
no-debug does not detect screen.debug() if screen is not last import in list

### ISSUE THREAD

**Issue by stefcameron** [testing-library/eslint-plugin-testing-library#174]
Love that `no-debug` rule, however:

```javascript
import { render, screen } from '@testing-library/react';
// OR just (but not very useful): import { screen } from '@testing-library/react';
...
screen.debug(); // IS detected as lint
```
where as
```javascript
import { screen, render } from '@testing-library/react';
...
screen.debug(); // is NOT detected as lint -- and should be
```

**Comment by Belco90** [comment]
Hey Stefan, thanks for creating the issue! Interesting one.

**Comment by stefcameron** [comment]
@Belco90 My pleasure! Thanks for the super quick fix! 😄

**Comment by stefcameron** [comment]
Wow, thanks! I'm honored! 🎉
